google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ClientSideStatementValueConverters.java

```diff
@@ -342,4 +342,19 @@ public Priority convert(String value) {
       return values.get("PRIORITY_" + value);
     }
   }
+
+  static class ExplainCommandConverter implements ClientSideStatementValueConverter<String> {
+    @Override
+    public Class<String> getParameterClass() {
+      return String.class;
+    }
+
+    @Override
+    public String convert(String value) {
+      if (value.length() <= 7) {
```

Comment:
added

Reply:
Consider a negative scenario like ```Explainselect 1```. Would it get converted to a successful query? If yes, we should add an additional check. I'm ok to have that as a follow up PR.

Reply:
"Explainselect 1" will give error "Unknown statement: explainselect 1" because it will not pass the regex check. And even ideally, it should not be converted to "Explain select 1".